### PR TITLE
Dev arcade

### PR DESCRIPTION
**Pull request by RetoStutz**

### CODE REVIEW

File: content/documentation/device-examples/components/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+| [LED Strip](/documentation/device-examples/components/ledstrip) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Potentiometer](/documentation/device-examples/components/potentiometer) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+| [Servo](/documentation/device-examples/components/servo) | Reto Stutz, Mike Schoder | [Example Devices](https://github.com/Pi4J/pi4j-example-components)|
+
+### Simple Implementation
+
+For a Simple Implementation, the recommendation is to use the [Launcher Class](https://github.com/Pi4J/pi4j-example-components/blob/Dev-Arcade/src/main/java/com/pi4j/example/Launcher.java).
+With this, a simple application can be started.
+
+TODO: better statement how the launcher works
```
Comment (FDelporte) [inline content/documentation/device-examples/components/_index.md:32]: Please remove TODO or even better add some extra info about the Launcher (with code example?)

File: content/documentation/device-examples/components/SimpleButton.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Simple Button
+weight: 200
+tags: ["Simple Button"]
+---
+### Description
+The [simplebutton](https://github.com/Pi4J/pi4j-example-components/tree/main/src/main/java/com/pi4j/components) (src/main/java/com/pi4j/components) is a template class, that you can use in your own Java-project.
```
Comment (FDelporte) [inline content/documentation/device-examples/components/SimpleButton.md:7]: Dead link. Probably still needs merge request in the example project?

**Review by FDelporte — COMMENTED** [review]
Great work! It's really great to have such a long list of nice examples to extend the documentation website.

File: content/documentation/device-examples/components/LEDStrip.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+delay(3000);
+
+//finishing and closing
+ledstrip.close();
+System.out.println("closing the app");
+System.out.println("Color "+ ledstrip.getPixelColor(0));
+```
+
+### Further application
+The class is implemented in the sample project [Potobooth](https://github.com/DieterHolz/PhotoBooth).
```
Comment (FDelporte) [inline content/documentation/device-examples/components/LEDStrip.md:67]: Typo Photobooth

File: content/documentation/device-examples/components/Servo.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+{{< figure link="/assets/documentation/device-examples/components/pictures/miniPowerSupplyModuleFrontBack.jpeg" caption="Mini Power Supply Module Front Back" caption-position="center" caption-effect="fade" >}}
+{{< /gallery >}}
+{{< load-photoswipe >}}
+
+### Code
+An example on how to use the Servo-Class from the [Hardware-Catalog](https://github.com/Pi4J/pi4j-example-components)
+
+```
+// Initialize servo motor component
+final var servoMotor = new ServoMotor(pi4j, PIN.PWM18.getPin());
```
Comment (FDelporte) [inline content/documentation/device-examples/components/Servo.md:30]: I see in preview, this code block doesn't format well. Maybe related to the final word here? 

Can you change ```   to ```java?

**Comment by FDelporte** [comment]
General remark: all java code blocks should start with triple-quote-java, see for example https://github.com/Pi4J/pi4j.github.io/blob/main/content/getting-started/minimal-example-application.md?plain=1#L144

File: content/documentation/device-examples/components/_index.md (hunk 1 above)
Comment (Mikscho) [reply content/documentation/device-examples/components/_index.md:32]: TODO is removed. How it is implemented, is shown in the example-apps

File: content/documentation/device-examples/components/SimpleButton.md (hunk 2 above)
Comment (Mikscho) [reply content/documentation/device-examples/components/SimpleButton.md:7]: yes, those links need the merge in the Example-Project to work.

File: content/documentation/device-examples/components/LEDStrip.md (hunk 3 above)
Comment (Mikscho) [reply content/documentation/device-examples/components/LEDStrip.md:67]: Typo is fixed in all components

File: content/documentation/device-examples/components/Servo.md (hunk 4 above)
Comment (Mikscho) [reply content/documentation/device-examples/components/Servo.md:30]: i formatted the code-blocks on all components

**Comment by Mikscho** [comment]
@FDelporte I fixed all open points. Could you please review it again, and then accept the pull-request?
The Example-Project (https://github.com/Pi4J/pi4j-example-components) is completed, and also needs a review.

**Pull request merged**